src/App.js

```diff
@@ -70,89 +70,78 @@ const Transition = styled(CSSTransition)`
   }
 `;
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.loadUserInfo();
-
-    this.onMenuToggle = this.onMenuToggle.bind(this);
-
-    this.state = {
-      overlay: false,
-    };
-  }
-
-  loadUserInfo() {
-    // Get user details from localStorage and save to react store
-    try {
-      var info = localStorage.getItem("userInfo");
-      if (info) {
-        let userInfo = JSON.parse(info);
-        console.log("Loaded UserId from storage : " + userInfo.userId);
-        console.log("Loaded token from storage : " + userInfo.token);
-        this.props.setCurrentUser({
-          userId: userInfo.userId,
-          token: userInfo.token,
-        });
-      } else {
-        console.log("user info not found");
-      }
-    } catch {
-      console.log("Cannot find info from localStorage");
+const loadUserInfo = (props) => {
+  // Get user details from localStorage and save to react store
+  try {
+    const info = localStorage.getItem("userInfo");
+    if (info) {
+      let userInfo = JSON.parse(info);
+      console.log("Loaded UserId from storage : " + userInfo.userId);
+      console.log("Loaded token from storage : " + userInfo.token);
+      props.setCurrentUser({
+        userId: userInfo.userId,
+        token: userInfo.token,
```

Comment:
why not destructure the userInfo object `const {userId , token } =. userInfo ` and here just use  props.setCurrentUser({
        userId, token})